doc/source/whatsnew/v0.25.0.rst

```diff
@@ -872,6 +872,7 @@ Other
 - Allow :class:`Index` and :class:`RangeIndex` to be passed to numpy ``min`` and ``max`` functions (:issue:`26125`)
 - Use actual class name in repr of empty objects of a ``Series`` subclass (:issue:`27001`).
 - Bug in :class:`DataFrame` where passing an object array of timezone-aware `datetime` objects would incorrectly raise ``ValueError`` (:issue:`13287`)
+- func:`read_table` has been undeprecated (:issue:`25220`)
```

Comment:
Maybe add a note in the v0.24.0 docs as well that the deprecation was removed in 0.25.0 for people reading that?

Reply:
Also, not sure where it first better, but I don't think the bugs section is were to put it. Maybe in the deprecations section, since it is most related to that?

Reply:
Good idea. I editing the whatsnew entry in v0.24.0 for the read_table deprecation to mention it was undeprecated in 0.25.0